src/templates/team.html

```diff
@@ -115,4 +115,4 @@ <h3>{{ template.name|raw }} ({{ template.fullname|raw }})
     data-item="{{ selectedItem }}"></div>
 <script src='app/js/tabs.min.js'></script>
 <script src='app/js/src/team.js'></script>
-{% endblock body %}
+{% endblock body %}
```

Comment:
Please add a newline character at the end of the file.

Reply:
Done! @NicolasCARPi